spec/controllers/users_controller_spec.rb

```diff
@@ -96,6 +103,15 @@
       end
     end
 
+    context 'with a member deactivate' do
+      it 'is redirected to select organization' do
+        login(deactivate_user)
+        get :index
+
+        assert_routing('/select_organization', controller: 'organizations', action: 'select_organization')
```

Comment:
Would be nice to use the `redirect_to` matcher here: https://relishapp.com/rspec/rspec-rails/docs/matchers/redirect-to-matcher

You can find some examples in our suite too.